Lib/Magnet/HotKey.swift

```diff
@@ -11,7 +11,7 @@
 import Cocoa
 import Carbon
 
-public final class HotKey: NSObject {
+public class HotKey: NSObject {
```

Comment:
Is there a use case to change `public` to `open` for mocking?

Reply:
Good point I'll make the change